src/intervals/arithmetic/absmax.jl

```diff
@@ -50,13 +50,27 @@ for f ∈ (:min, :max)
             isempty_interval(y) && return y
             return _unsafe_bareinterval(T, $f(inf(x), inf(y)), $f(sup(x), sup(y)))
         end
+        function Base.$f(x::BareInterval{T}, y::Rational) where {T<:NumTypes}
+            isempty_interval(x) && return x
+            return _unsafe_bareinterval(T, $f(inf(x), y), $f(sup(x), y))
+        end
```

Comment:
We purposely do not allow functions mixing `BareInterval` and `Number` in their computations, because `BareInterval` cannot keep track of guarantees ("NG" flag)or decorations.
So my first impulse here would be not to define such function.